src/components/Accounts/Actions/FeeFineActions.js

```diff
@@ -406,6 +383,7 @@ class Actions extends React.Component {
     body.comments = this.assembleTagInfo(values);
     body.servicePointId = servicePointId;
     body.userName = `${lastName}, ${firstName}`;
+    body.transactionInfo = values.transaction || '-';
```

Comment:
`transactionInfo` should not be here, this field accepted only for `payment` action. It should be removed in `onSubmit Many` or check for action type should be added

Reply:
Done